Modules/_io/bufferedio.c

```diff
@@ -362,16 +362,24 @@ _enter_buffered_busy(buffered *self)
     }
 
 #define IS_CLOSED(self) \
-    (!self->buffer || \
+    (!self->buffer ? !self->buffer : \
```

Comment:
```suggestion
    (!self->buffer ? 1 : \
```

To me the double `!self->buffer` is hard to comprehend. This makes it more obvious "buffer was unset; so return the file is closed"